scripts/utils.ts

```diff
@@ -84,7 +86,20 @@ export async function setupApis(
   // is necessary for generating the SDK (as part of the user agent header).
   // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
   apis.metadata.sdkVersion = await readJsonSync(PACKAGE_JSON).version;
-  await apis.init();
+
+  // Editing the name of the Shopper Context API Model so our name is used for generating class name.
+  // This is hard-coded for now but in the future if we handle the case where type name and title clash better, this can be removed.
+  const shopperContextApi = apis.children.find(
+    api => api.name.original === 'shopper-context'
+  );
+
+  if (shopperContextApi) {
+    shopperContextApi.name = new Name('shopper-contexts');
+  }
+
+  // We are calling the init for children (which will call loadModel) since we can pass the updateName parameter as false (default was true).
+  // With this way the ApiModel doesn't change its name again from the loaded api model, which will use the raml title field.
+  await Promise.all(apis.children.map(api => (api as ApiModel).init(false)));
```

Comment:
what  happens if you don't reload the model?

Reply:
you are calling` .init(false) `for all the APIs while we need it only shopper-context. Can we avoid that as there is a possibility to break customer implementations

Reply:
I can totally try to call with (false) just for Shopper Context. I don't think this current version can break the changes now because the titles are consistent with the download names. I do agree it is safer to call this just for Shopper Context.